overlay/overlay.proto

```diff
@@ -96,6 +96,10 @@ message AgentInfo {
 
   // The overlay networks that exist on this agent.
   repeated AgentOverlayInfo overlays = 2;
+
+  // Number of configuration attempts. For more information
+  // please see max_configuration_attempts in AgentConfig.
+  optional uint32 configuration_attempts = 3;
```

Comment:
Why do we want this default? The default should be set only for `max_configuration_attemtps`. I think this field gives the state of the number of configuration attempts so it should have a value to start with?

Reply:
I don't get it. `configuration_attempts` doesn't have default value. it doesn't have any value if it's not set.

Reply:
My bad !! Was reading this wrong !!